### PR TITLE
Added git information for anonymous authentication

### PR DESCRIPTION
**Pull request by milindk8**
# Description

Git account: Need to update documentation for authentication type "Anonymous" while adding the Version controller

Fixes - https://github.com/devtron-labs/devtron/issues/2150

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

By selecting git account having public account

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by sohel-rp — APPROVED** [review]

File: src/components/gitProvider/GitProvider.tsx [hunk 1]
```diff
@@ -589,18 +591,24 @@ function GitForm({
...
                     ))}
                 </div>
-                <div className="flex fs-12 left pt-4 mb-20" style={{ color: '#6b778c' }}>
+                <div className="flex fs-12 left pt-4 mb-16" style={{ color: '#6b778c' }}>
                     <Warn className="icon-dim-16 mr-4 " />
                     Once configured, authentication type cannot be switched from HTTPS (user auth/anonymous) to SSH or
                     vice versa.
                 </div>
+                {state.auth.value === AuthenticationType.ANONYMOUS && (
+                    <div className="pt-10 pb-10 pl-16 pr-16 fs-13 br-4 bw-1 bcb-1 lh-20 cn-9 w-100 eb-2 flexbox mb-40">
```
Comment (shivani170) [inline src/components/gitProvider/GitProvider.tsx:607]: we have one common component ÌnfoColourBar`to show this type of information. Please reuse that

File: src/components/material/MaterialView.tsx [hunk 2]
```diff
@@ -183,7 +185,13 @@ export class MaterialView extends Component<MaterialViewProps, MaterialViewState
                 </div>
 
             </div>
-
+            {this.props.material.gitProvider?.authMode === AuthenticationType.ANONYMOUS && (
+                <div className="pt-10 pb-10 pl-16 pr-16 fs-13 br-4 bw-1 bcb-1 lh-20 w-100 eb-2 cn-9 flexbox mb-20">
```
Comment (shivani170) [inline src/components/material/MaterialView.tsx:189]: Same here

File: src/components/gitProvider/GitProvider.tsx (hunk 1 above)
Comment (milindk8) [reply src/components/gitProvider/GitProvider.tsx]: Added

File: src/components/material/MaterialView.tsx (hunk 2 above)
Comment (milindk8) [reply src/components/material/MaterialView.tsx]: done

**Review by shivani170 — APPROVED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**